ambry-router/src/main/java/com/github/ambry/router/SimpleOperationTracker.java

```diff
@@ -191,19 +207,24 @@
     for (ReplicaId replicaId : replicas) {
       examinedReplicas.add(replicaId);
       String replicaDcName = replicaId.getDataNodeId().getDatacenterName();
-      if (replicaDcName.equals(originatingDcName)) {
+      boolean isLocalDcReplica = replicaDcName.equals(datacenterName);
+      boolean isOriginatingDcReplica = replicaDcName.equals(originatingDcName);
+      if (isLocalDcReplica) {
+        numReplicasInLocalDc++;
```

Comment:
minor: `numReplicasInLocalDc` is updated but never accessed. Consider removing it?

Reply:
Sorry left that in there since I was thinking about adding in some changes to the logic for adding backup replicas when includeOriginating is false to just use the total number of replicas in the local and remote dcs instead of hardcoding it to 6. I ultimately decided not to couple these changes though since it affects our current production cluster the most.